packages/ts-client/src/webrtc/internal.ts

```diff
@@ -70,3 +70,27 @@ export type EndpointWithTrackContext<EndpointMetadata, TrackMetadata> = Omit<
 > & {
   tracks: Map<string, TrackContextImpl<EndpointMetadata, TrackMetadata>>;
 };
+
+export const mapMediaEventTracksToTrackContextImpl = <
```

Comment:
lets make it more like Java
```suggestion
export const mapMediaEventTracksToTrackContextImplFactory = <
```